### ISSUE TITLE
Doubt regarding the validation script

### ISSUE THREAD

**Issue by zackdilan** [huggre/biometric_authentication_with_iota#1]
Hello @huggre , 
First of all great effort!! I have been following your tutorials and have a doubt regarding the validation script. 
In the doc, you have mentioned, the validation script downloads the entire employee's fingerprint data from the tangle and stores it internally on the reader. and you have mentioned each employee has a unique IOTA seed.

As far I have understood, the validation script asks for the employee seed. and tries to get the account info. So basically the script has only info related to that particular seed (not the entire employees).  And you need the different employees seed to get their account info. then you can have the local database on the reader. 

I hope you might have understood my confusion in the code as well as the doc. Please feel free to explain if I m wrong.

**Comment by huggre** [comment]
Hi, nice to hear that you like my tutorials :-)
Not sure I totally understand your question but the idea here is that we only download the fingerprints for one employee at the time to the reader, not all employees.
So each time we want to validate an employee we enter his/her seed to find the address where the fingerprint bundle is stored, and then downloads it to the reader before scanning and validating. For the next employee we perform the same process just with a different employee/seed.
In other words, the reader only holds the fingerprints for one employee at the time.
Hope that helps.
